tests/unit/ResolutionDates.spec.ts

```diff
@@ -165,8 +211,9 @@ describe('Resolution Dates component - review mode', () => {
   let wrapperFactory: any
 
   beforeAll(() => {
+    store.state.stateModel.shareStructureStep.shareClasses = shareClasses
     wrapperFactory = (propsData: any) => {
-      return mount(ResolutionDates, { propsData: { ...propsData, isEditMode: false }, vuetify })
+      return mount(ResolutionDates, { propsData: { ...propsData, isEditMode: false }, vuetify, store })
```

Comment:
Do we want to have a test for is resolutions dates valid

Reply:
We do, absolutely. I will expand more on the unit tests in the next commit as i'm sure there will be more to add. 
The intention here was to get the validity code into dev, because it has crossover with the larger validators ticket.

Reply:
10-4

Reply:
Updated!